test/videojs-contrib-hls.test.js

```diff
@@ -1531,13 +1532,14 @@ QUnit.test('playlist blacklisting duration is set through options', function(ass
   assert.equal(this.env.log.warn.args[0],
               'Problem encountered with the current HLS playlist. HLS playlist request error at URL: media.m3u8 Switching to another playlist.',
               'log generic error message');
+  // this takes one millisecond
```

Comment:
If the new response didn't be added here, it will error for the second(final) rendition. Since we clear blacklist duration for the first rendition when final rendition errors, we can't test if the blacklist duration is set through option successfully.